src/python/pants/backend/go/subsystems/gotest.py

```diff
@@ -24,3 +28,34 @@ class GoTestSubsystem(Subsystem):
         ),
         passthrough=True,
     )
+
+    coverage_mode = EnumOption(
+        "--cover-mode",
```

Comment:
Bike shed: Should this be `--cover-mode` which maps to what Go calls it or `--coverage-mode` which is more descriptive and maps to what Pants uses for coverage-related terms?

Reply:
I used "coverage" internally for a lot of variable names, but Go consistently uses "cover" instead of "coverage."